### PR TITLE
Pattyssilveira/modal window component

### PR DESCRIPTION
**Pull request by pattyssilveira**
This PR adds : 
- Modal window
- Modal header
- Modal body
- Modal footer
- Close button 
- Full-width button

### CODE REVIEW

File: src/components/atoms/modal-header/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames/bind'
+
+// import CloseIcon from '_atoms/close-icon'
```
Comment (dleitee) [inline src/components/atoms/modal-header/index.js]: ?

**Review by dleitee — CHANGES_REQUESTED** [review]

File: src/components/atoms/modal-body/styles.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.modalBody {
+  display: flex;
+}
+
+.modalText {
+    align-self: flex-start;
+    font-family: 'Source Sans Pro', sans-serif;
+    font-size: 13px;
+    font-weight: 200;
+    color: #666666;
```
Comment (beatriz1304) [inline src/components/atoms/modal-body/styles.css]: Put that inside ``css-variables.js`` file

File: src/components/atoms/modal-footer/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames/bind'
```
Comment (beatriz1304) [inline src/components/atoms/modal-footer/index.js]: Remove the bind it's not necessary 
``import classNames from 'classnames'``

File: src/components/atoms/modal/index.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames/bind'
+import ReactModal from 'react-modal'
+
+import styles from './styles.css'
+
+class Modal extends React.Component {
```
Comment (beatriz1304) [inline src/components/atoms/modal/index.js]: You can import `Component` from react lib and then just you here

`class Modal extends Component`

File: src/components/atoms/modal/index.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```
Comment (beatriz1304) [inline src/components/atoms/modal/index.js]: import here the `Component` class

`import React, {Component} from 'react' `

File: src/components/atoms/modal/index.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    children: PropTypes.node.isRequired,
+  }
+
+  static defaultProps = {
+    iconURL: undefined,
+    className: undefined,
+    onClick: () => {},
+  }
+
+  constructor() {
```
Comment (beatriz1304) [inline src/components/atoms/modal/index.js]: With ES6 it's not necessary use the constructor and calling the super

Comment (beatriz1304) [reply src/components/atoms/modal/index.js]: You can just declare your state like:
`state = {modalIsOpen: true}`

**Pull request closed without merging**